lib/bootstrap_form/form_builder.rb

```diff
@@ -52,7 +52,9 @@ def initialize(object_name, object, template, options)
       define_method(with_method_name) do |name, options = {}, html_options = {}|
         prevent_prepend_and_append!(options)
         form_group_builder(name, options, html_options) do
-          content_tag(:div, send(without_method_name, name, options, html_options), class: control_specific_class(method_name))
+          html_class = control_specific_class(method_name)
+          html_class = "#{html_class} form-inline" unless options[:skip_inline]
```

Comment:
Is the `:skip_inline` option a new option you're proposing? If so, could you please update the CHANGELOG to note the new functionality, and probably add a description of the option in the `README`.